docs/user_guide/faq.rst

```diff
@@ -0,0 +1,112 @@
+    whether any use of ZIP code was as a proxy for race, or for some other reason.
+
+The model is unfair because the data are biased. Isn't it better to get better data?
+    If a model is unfair due to biased data (note that this is not the only reason a model
+    may be unfair), then obtaining unbiased data is definitely the best solution.
+    However, doing so might not always be practical.
+    In these cases, the mitigation techniques in Fairlearn and similar packages can redistribute
+    harms between subgroups in the data.
+    The extent to which this is 'fair' will depend on how the harms are redistributed, and the
+    broader context of the system of which the model is a part.
+
+Won't making a model fairer reduce its accuracy?
+    Any technique which reduces disparity in some metric is most likely to impact other metrics.
+    However, we have found empirically that it is usually possible to obtain substantial reductions
+    in disparity (as measured by some suitable metric) at minimal cost in accuracy.
+    Indeed, for certain subgroups in the data, accuracy may be improved dramatically.
+
+Fairlearn questions
+-------------------
+
+Can Fairlearn make my model fair?
+    Naive use of Fairlearn is not going to result in fairer models.
+    Indeed, when used incorrectly, Fairlearn could produce a model which is *less* fair.
+
+    What Fairlearn can do is generate a model which has lower disparity as measured by
+    a user-specified metric.
+    Whether this makes the model fairer is outside the remit of Fairlearn - or indeed of
+    machine learning in general.
+    To determine whether the new model is fairer, the whole system of which it is a part
+    needs to be considered in terms of the societal context in which it is used.
+    See, for example, "Fairness and Abstraction in Sociotechnical Systems" by Selbst et al.,
+    listed on our :ref:`further_reading` page. 
+
+What sort of fairness related harms can Fairlearn address?
+    Fairlean concentrates on group fairness.
+    That is, obtaining the minimum disparity on some metric evaluated on
+    specified subgroups in the data.
+    This is certainly not the only way of looking at fairness.
+    For example, there is the concept of individual fairness (where decisions are evaluated
+    on the level of individuals rather than groups) and counter-factual fairness (e.g. does the
+    decision change if an individual's gender is changed from male to female).
+    See :ref:`types_of_harms` for more information.
+    Beyond these, there are also fairness concepts which are note amenable to
+    mathematical (and hence algorithmic) expression - justice, due process and
+    righting historic iniquities for example.
+
+Can Fairlearn be used to detect bias in datasets?
+    We do not have concrete plans for this at the present time.
+
+Can Fairlearn recommend ways to make my model fairer?
+    No. Fairlearn is not magic.
```

Comment:
I'm not sure we want to be snarky in the responses....

It's actually a good question, and recommendations *could* theoretically be made. They may just not be very good unless we highly contextualize them, and that's a lot of work with very uncertain outcomes as far as I see it. Better to provide thorough examples.

This could link to the fairness checklist http://www.jennwv.com/papers/checklists.pdf